internal/testutils/bubblewrap.go

```diff
@@ -0,0 +1,152 @@
+package testutils
```

Comment:
I had some trouble understanding why b327f0b6abcb6527d685a014250ffddae5b9a1e2 was necessary. At first it looked like it just adds quite a lot of complexity for which there is no need. I think I now understand that the reason is that we also need to call `lckpwdf` in an environment which allows it to write to `/etc`. Can you please update the commit message to explain that?

Reply:
> I think I now understand that the reason is that we also need to call lckpwdf in an environment which allows it to write to /etc

Yes, plus there are various other reasons why that has happened:
 - Be able to easily add and run multiple tests inside the bwrap environment
 - Have a generic way to test inside a such environment (that will be useful in future for other integration tests)
 - Run all these tests in parallel

Added in the commit

Reply:
>  I think I now understand that the reason is that we also need to call lckpwdf in an environment which allows it to write to /etc.

I still don't see that mentioned in the commit message.

Reply:
Oh, ok... I'll add it just before the final rebase.